argilla-server/tests/unit/contexts/hub/test_hub_dataset.py

```diff
@@ -270,6 +271,31 @@ async def test_hub_dataset_import_to_with_class_label_fields_using_no_label(
         record = (await db.execute(select(Record))).scalar_one()
         assert "label" not in record.fields
 
+    async def test_hub_dataset_import_to_with_chat_fields(self, db: AsyncSession, mock_search_engine: SearchEngine):
+        dataset = await DatasetFactory.create(status=DatasetStatus.ready)
+
+        await ChatFieldFactory.create(name="messages", required=True, dataset=dataset)
+
+        await dataset.awaitable_attrs.fields
+        await dataset.awaitable_attrs.questions
+        await dataset.awaitable_attrs.metadata_properties
+
+        hub_dataset = HubDataset(
+            name="mlabonne/ultrachat_200k_sft",
+            subset="default",
+            split="train_sft",
+            mapping=HubDatasetMapping(
+                fields=[
+                    HubDatasetMappingItem(source="messages", target="messages"),
+                ],
+            ),
+        )
+
+        await hub_dataset.take(1).import_to(db, mock_search_engine, dataset)
+
+        record = (await db.execute(select(Record))).scalar_one()
+        assert record.fields["messages"]
```

Comment:
I'm not testing the content because is huge. Knowing that the import is working without raising any exception and that the field is there is enough I guess.